core/src/worker/api.ts

```diff
@@ -146,7 +146,8 @@ export async function getAggregator({
  */
 export async function storeErrorMsg({ data, logger }: { data: IErrorMsgData; logger: Logger }) {
   try {
-    const error = (await axios.post(ERROR_ENDPOINT, data))?.data
+    const response = (await axios.post(ERROR_ENDPOINT, data))?.data
```

Comment:
I think we don't have to return the "response". It just tries to pass data somewhere, there is nothing to do with them afterward.